### PR TITLE
 Implement Pull #3306 from citra: citra_qt: Drop Qt 5 version checks in code

### PR DESCRIPTION
**Pull request by N00byKing**

### CODE REVIEW

**Comment by bunnei** [comment]
Travis has had an outage all day, going to wait until it's back online (hopefully tomorrow) before merging this one, so we can verify all platforms. Thanks!

File: src/yuzu/bootmanager.cpp [hunk 1]
```diff
@@ -120,15 +116,14 @@ GRenderWindow::~GRenderWindow() {
...
-#if QT_VERSION > QT_VERSION_CHECK(5, 0, 0)
+
     // If the thread started running, move the GL Context to the new thread. Otherwise, move it
     // back.
     auto thread = (QThread::currentThread() == qApp->thread() && emu_thread != nullptr)
                       ? emu_thread
                       : qApp->thread();
     child->context()->moveToThread(thread);
-#endif
+
```
Comment (bunnei) [inline src/yuzu/bootmanager.cpp]: This line breaks the clang-format check. Can you please install clang-format and run it on this file? Thank you!

Comment (N00byKing) [reply src/yuzu/bootmanager.cpp]: Hope d02e966 fixed it

**Comment by bunnei** [comment]
👍 ty lgtm

**Pull request merged**